app/javascript/mastodon/reducers/markers.ts

```diff
@@ -0,0 +1,23 @@
+import { createAction, createReducer } from '@reduxjs/toolkit';
+
+import { submitMarkersAction } from 'mastodon/actions/markers';
+
+const initialState = {
+  home: '0',
+  notifications: '0',
+};
+
+export const markersSubmitSuccess = createAction<{
+  homeValue?: string;
+  notificationsValue?: string;
+}>('markers/success');
```

Comment:
Why define an action in `mastodon/reducers/`? Also, is it used anywhere?

Reply:
This was a leftover and has been replaced by `submitMarkersAction.fulfilled`